lib/goodcity/health_checks/package_stockit_id_nil_check.rb

```diff
@@ -0,0 +1,19 @@
+require 'goodcity/health_checks/base'
+
+module Goodcity
+  class HealthChecks
+
+    class PackageStockitIdNilCheck < Base
+      desc "Packages should contain a stockit_id reference."
+      def run
+        ids = Package.where(stockit_id: nil).pluck(:id)
+        if ids.count == 0
+          pass!
+        else
+          fail_with_message!("GoodCity Packages with nil stockit_id: #{ids.join(', ')}")
+        end
+      end
```

Comment:
Decided to back out of these changes for now... the failure message that is passed in required access to the query to list the ids so not possible to defer execution of query till later.